src/nvim/api/vim.c

```diff
@@ -54,6 +54,58 @@ void nvim_command(String command, Error *err)
   try_end(err);
 }
 
+/// Retrieves highlight description from its name
+///
+/// @param name Highlight group name
+/// @return a highlight description in dictionary {'hl_id': X, 'bg': Y, 'fg': Z}
+/// @see nvim_hl_from_id
+Dictionary nvim_hl_from_name(String name, Error *err)
```

Comment:
`nvim_{thing}` should be preferably reserved for rpc objects like buffer. `nvim_get_hl_by_name` is better.